### ISSUE TITLE
Why is my eval_accuracy very low?

### ISSUE THREAD

**Issue by wangyunxiangsha** [brightmart/sentiment_analysis_fine_grain#6]
eval_accuracy = 0.05
eval_loss = 0.1677554
global_step = 78750
loss = 0.16775554

**Comment by leejason** [comment]
Me too. Any advice would be appreciated.

**Comment by minmummax** [comment]
me too

**Comment by bbingy** [comment]
Me too. Does anyone find the problem?

**Comment by XSilverBullet** [comment]
Me too.  Can anyone answer this question?

**Comment by netsesame2** [comment]
I got the similar result after nearly 16 hours run on GPU.  
Can't accept both the accuracy and the long time, is it normally or not?

**Comment by YuMiaoTHU** [comment]
Me too！！！！anyone can help? Thanks!!!

**Comment by qiu-nian** [comment]
我的accuracy也是0.05，有谁知道怎么解决这个问题吗？是不是代码或者计算的问题啊

**Comment by YuMiaoTHU** [comment]
> 我的accuracy也是0.05，有谁知道怎么解决这个问题吗？是不是代码或者计算的问题啊

acc的计算不适用于多标签，低也是正常的，你可以自己写一个文件用来计算precision，recall，F1值

**Comment by qiu-nian** [comment]
嗯嗯，好的，多谢您的回复，我也正在看多标签分类的一些指标的写法，谢谢您的解答，不然感觉自己可能在这个问题上饶好久，真的的太感谢了

**Comment by qiu-nian** [comment]
> > 我的accuracy也是0.05，有谁知道怎么解决这个问题吗？是不是代码或者计算的问题啊
> 
> acc的计算不适用于多标签，低也是正常的，你可以自己写一个文件用来计算precision，recall，F1值

我后面又找到一个计算每个标签的acc的代码了，然后试了一下是可以的，https://www.kaggle.com/javaidnabi/toxic-comment-classification-using-bert/  就是这个项目。
所以大家的acc都是0.05应该是因为作者的acc那一块的代码写得有问题。。。。
